sklearn/linear_model/coordinate_descent.py

```diff
@@ -1084,7 +1084,10 @@ def fit(self, X, y):
             to avoid unnecessary memory duplication. If y is mono-output,
             X can be sparse.
 
-        y : array-like, shape (n_samples,) or (n_samples, n_targets)
+        y : array-like,
+              - shape (n_samples,), valid for `ElasticNetCV` and `LassoCV`.
+              - shape (n_samples, n_tasks), valid for `MultiTaskElasticNetCV` 
+                and `MultitaskLassoCV`.
```

Comment:
this renders fine:

https://50700-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.linear_model.ElasticNetCV.html#sklearn.linear_model.ElasticNetCV.fit

but it's not great that docstring of ElasticNetCV mentions estimators which are unrelated.... I am tempted to implement the fit in ElasticNetCV that will call super().fit just to have a good docstring...

Reply:
wouldn't we also need to implement fit methods in LassoCV, MultiTaskElasticNetCV and MultitaskLassoCV just for the doctring?